espnet2/bin/enh_scoring.py

```diff
@@ -166,7 +167,7 @@ def get_parser():
     )
     group.add_argument("--key_file", type=str)
     group.add_argument("--ref_channel", type=int, default=0)
-    group.add_argument("--flexible_numspk", type=bool, default=False)
+    group.add_argument("--flexible_numspk", default=False, action="store_true")
```

Comment:
Setting the type to `str2bool` seems a better option as we can intuitive show how to use `--flexible_numspk` by passing `True` or `False`.

Reply:
sounds good. I'll update it.